cloudbuild.yaml

```diff
@@ -7,14 +7,53 @@ timeout: 1500s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211118-2f2d816b90'
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-ad65926f6b'
+    id: image-amd64
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+    - VERSION=$_GIT_TAG
+    - BASE_REF=$_PULL_BASE_REF
+    args:
+    - image.amd64
+    waitFor: ['-']  # run in parallel
+
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-ad65926f6b'
+    id: image-arm
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+    - VERSION=$_GIT_TAG
+    - BASE_REF=$_PULL_BASE_REF
+    args:
+    - image.arm
+    waitFor: ['-']
+
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-ad65926f6b'
+    id: image-arm64
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+    - VERSION=$_GIT_TAG
+    - BASE_REF=$_PULL_BASE_REF
+    args:
+    - image.arm64
+    waitFor: ['-']
+
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-ad65926f6b'
     entrypoint: make
     env:
     - DOCKER_CLI_EXPERIMENTAL=enabled
     - VERSION=$_GIT_TAG
     - BASE_REF=$_PULL_BASE_REF
     args:
     - push-all
```

Comment:
this is still going to call all 3 builds in this step: https://github.com/kubernetes-sigs/descheduler/blob/9f6afd4d2526cc31e1486d25e2f1ecd1d7476de5/Makefile#L87

maybe we need a new `make` target that decouples `push` from `build`?

Reply:
true, when it tries to build them it should go quicker since there won't be any changes. but I think it would be better to not make an unnecessary `build` call when we really just need a push. it makes it clearer what each step is doing and keeps job logs cleaner for debugging. wdyt?

Reply:
that's fair, should be coming right up!